frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -112,6 +117,43 @@ function FileManagerMenu:setUpdateItemTable()
     end
 
     -- tools tab
+    if Device.isKobo() then
```

Comment:
I have a suggestion, instead of adding too many if Device.isKobo or similar if-else in different files, should we move these logic into device?
For this specific case, we can add a initFileManagerMenu function in device, and override in kobo's device implementation.